docs/blog/newsletter-february-2018.rst

```diff
@@ -0,0 +1,121 @@
+
+.. post:: February 5, 2018
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – February 2018
+#########################################
+
+Hi everybody! Kelly O. here – back in the saddle after getting through the whole *OMG, it's a different year now* transition. 
+I'm excited to report that we have a TON of stuff happening in the community. 
+I guess we're starting this new year off strong!
+
+First things first, the march toward our Portland conference continues at a steady pace. 
+May 6-8 is starting to feel *awfully* close, and we couldn't be more excited. 
+We're working on confirming our last few speakers. 
+You'll hear from us with the full speaker line-up in the next week or so.
+
+*NOTE: If you're not subscribed to the North American conference mailing list,* 
+`you can sign up here <http://www.writethedocs.org/newsletter/>`_.
```

Comment:
I would link only `sign up here` and `convince your manager` page, but that might just be a personal thing